io/disk/usb_tests.py.data/usb_tests.yaml

```diff
@@ -0,0 +1,4 @@
+disk: "/dev/disk/by-path/pci-0154:80:00.0-usb-0:4:1.0-scsi-0:0:0:0"
+pci_device: "0154:80:00.0"
```

Comment:
remove above 2 parameter values, add them in Readme as example

Reply:
Yes, Will remove and add them to readme